lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -722,3 +729,62 @@ hex = convertToBase Base16
 -- | Decode a 'ByteString' from base16
 fromHex :: ByteArray bout => ByteString -> Either String bout
 fromHex = convertFromBase Base16
+
+data ErrUnXPrvStripPub
+    = ErrNoRoundtripMismatch
```

Comment:
Seems like a double negative. Maybe just `ErrNoRoundtrip` instead.

Reply:
`ErrNoRoundtrip` feels like something called "Roundtrip" is missing, like you would say "ErrNoPassphrase" for a missing field. 

Perhaps: `ErrCannotRoundtrip` or `ErrRoundtripFailure` ?